fastlane/actions/gitlab_create_project.rb

```diff
@@ -10,13 +10,17 @@ def self.run(params)
         source = Gitlab.client(endpoint: params[:endpoint_src], private_token: params[:api_token_src])
         destination = Gitlab.client(endpoint: params[:endpoint_dst], private_token: params[:api_token_dst])
         original_project = params[:project]
+        if original_project.owner
```

Comment:
Oh yeah, you are right and we actually had a few projects where this was the case!
So i guess this is fine 👍